gammapy/time/models.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
At the very top of each file in Gammapy we have the license line. Please add it.